app/javascript/packages/document-capture/components/documents-step.jsx

```diff
@@ -42,38 +45,60 @@ function DocumentsStep({
   const { notReadySectionEnabled, exitQuestionSectionEnabled, selfieCaptureEnabled } =
     useContext(FeatureFlagContext);
 
-  /**
-   * Sides of document to present as file input.
-   *
-   * @type {DocumentSide[]}
-   */
-  const documentSides = selfieCaptureEnabled ? ['front', 'back', 'selfie'] : ['front', 'back'];
+  const pageHeaderText = selfieCaptureEnabled
+    ? t('doc_auth.headings.document_capture_with_selfie')
+    : t('doc_auth.headings.document_capture');
+
+  const DocumentTipList = withProps({
```

Comment:
Same coment as with `DocumentFront`, this should be outside the `DocumentsStep` component to avoid later problems figuring out which props are actually used.